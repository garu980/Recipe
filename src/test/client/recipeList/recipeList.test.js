import { shallow } from 'enzyme';
import * as React from 'react';
import RecipeList from '../../../client/presentation/recipeList/recipeList';
import * as Redux from 'react-redux';

describe('Recipe list rendering', () => {
  it('renders the component', () => {
    const recipeList = shallow(<RecipeList />);
    expect(recipeList).toMatchSnapshot();
  });
});
