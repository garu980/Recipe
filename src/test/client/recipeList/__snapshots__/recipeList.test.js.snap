// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe list rendering renders the component 1`] = `ShallowWrapper {}`;
